source/solvers/vof_assemblers.cc

```diff
@@ -198,23 +200,21 @@ VOFAssemblerCore<dim>::assemble_rhs(VOFScratchData<dim> &      scratch_data,
 
       // Implementation of a DCDD shock capturing scheme.
       // For more information see
-      // Tezduyar, T. E., & Park, Y. J. (1986). Discontinuity-capturing
-      // finite element formulations for nonlinear
-      // convection-diffusion-reaction equations. Computer methods in
-      // applied mechanics and engineering, 59(3), 307-325.
-
-      // Gather the order of the VOF interpolation
-      const double order = this->fem_parameters.VOF_order;
-
-      // Calculate the artificial viscosity of the shock capture
-      const double vdcdd = (0.5 * h) * (velocity.norm() * velocity.norm()) *
-                           pow(phase_gradient.norm() * h, order);
+      // Tezduyar, T. E. (2003). Computation of moving boundaries and interfaces
+      // and stabilization parameters. International Journal for Numerical
+      // Methods in Fluids, 43(5), 555-575. We implement the equivalent of
+      // equation (70) and (79)
 
       const double tolerance = 1e-12;
 
+      // In Tezduyar 2003 this is noted r
       Tensor<1, dim> gradient_unit_vector =
         phase_gradient / (phase_gradient_norm + tolerance);
 
+      // Calculate the artificial viscosity of the shock capture
+      const double vdcdd =
+        (0.5 * h * h) * velocity.norm() * phase_gradient_norm;
+
       // We neglect to remove the diffusion aligned with the velocity
       // as is done in the original article. This term generates poorer
       // results than just using the vdcdd approach.
```

Comment:
Is this still true?

Reply:
Yup, will be the next step in a next PR

Reply:
In that case,  is this more accurate? 
```suggestion
      // We neglect to remove the diffusion aligned with the velocity
      // as is done in the original article.
```